docs/eigenlayer/operator-guides/key-management/intro-key-management.md

```diff
@@ -9,21 +9,38 @@ This section presents key management suggestions for operators. Within the conte
+Focusing on Ethereum validator keys exemplifies keys necessitating optimal security and accessibility measures, as they may maintain a robust connection with Nodes, further underlining their importance. Indeed, in Ethereum, a validator's identity is synonymous with their public key. To be precise, each validator possesses two sets: a signing and withdrawal keys.
 
-### Signing keys
+#### Signing keys
 
 A *signing key* is the key a validator needs to sign attestations and propose blocks. Because a validator needs to sign a message at least once per epoch, the client software must have custody of the key.
 
-### Withdrawal keys
+#### Withdrawal keys
 
 Because the client software is always connected to the internet, there is a chance that one’s signing key is compromised. To reduce the impact of such a breach, the actions a validator can perform are split between two keys.
 
 The signing key, as explained above, is used for the validator to perform their duties. On the other hand, the *withdrawal key* has the power to control a validator's funds (transferring and withdrawing ETH).
 
 A validator should only need to use their withdrawal keys a few times over the lifetime of being a validator. This means they can be put into cold storage and stored with high security (offline).
 
+### Eigenlayer
+
+Eigenlayer works differently from Ethereum due to it being natively a delegated PoS system. This means operators don't need to custody a separate key for each 32 Eth; they only need a single key per eigenlayer operator. This does imply that proper key management tools are even more important.
+
+Contrary to Eigenlayer, where an operator controls a bunch of (signing, withdrawal) key pairs, an eigenlayer operator controls a single operator key, as well as a separate key (or a small number of keys) for each AVS they are registered in.
+
+#### Operator keys
+
+An operator's operator key is the ecdsa key that is used to authenticate any interaction with the eigenlayer core contracts, and hence control an operator's actions such as registering to an operator, changing its operator parameters, force undelegating a staker, etc.
+
+The operator key should NEVER be used as an AVS key. They should also only be used to interact with the eigenlayer core contracts via the eigenlayer-cli or other operator-built tools, and should NOT be loaded into any node software. Any action that needs to be programatically triggered on the AVS contracts should be authorized via an AVS key, not the operator key.
+
+#### AVS signing keys
+
+AVS keys are actively used by node software to sign messages for AVSs. AVSs that use the eigenlayer-middleware contracts would typically use a bn254 key as the avs signing key, but other AVSs might require an ecdsa, ed25519, or any other key type.
+
+We advise operators to use a different key for every AVS, and NEVER to reuse their operator key as an AVS signing key. 
```

Comment:
I think there's a way to highlight it. like a `info` or `warn` block

we could also say. We strongly advise...  to be give very very strong advice

Reply:
how about this? [b16685e](https://github.com/Layr-Labs/eigenlayer-docs/pull/218/commits/b16685e84116eb3c33b4f30f54345d496e279ec7)